### PR TITLE
Adds vagrant instructions

### PR DESCRIPTION
**Pull request by spuder**
Adds instructions for spinning up a vagrant vm for testing. 
Completes https://github.com/home-assistant/home-assistant/pull/2112

### CODE REVIEW

File: source/getting-started/installation-vagrant.markdown [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+</p>
+
+Download the home-assistant source code by either downloading the .zip file from [github releases page](https://github.com/home-assistant/home-assistant/releases), or by using [git](https://git-scm.com/)
+
+    git clone https://github.com/home-assistant/home-assistant.git
+    vagrant up
+
+This will launch an Ubuntu 14.04 VM using Virtualbox. Home-Assistant will then be available at the following ip:port
+
+http://192.168.33.10:8123
```
Comment (shaftoe) [inline source/getting-started/installation-vagrant.markdown]: why not http://localhost:8123
Is as easy as having `config.vm.network "forwarded_port", guest: 8123, host: 8123` in the Vagrantfile

Comment (spuder) [inline source/getting-started/installation-vagrant.markdown]: Yes, I agree that will probably be better. Will add that when back on my normal computer later today.

**Comment by spuder** [comment]
Closing since this will be covered in a different PR. See https://github.com/home-assistant/home-assistant/pull/2112#issuecomment-222283334

**Comment by shaftoe** [comment]
Great job, thanks for pointing this out, I'll fork this and adapt to https://github.com/home-assistant/home-assistant/pull/2112#issuecomment-222283334

**Pull request closed without merging**